settings.gradle

```diff
@@ -13,6 +13,7 @@ rootProject.name = 'sptribs-case-api'
 
 
 //includeBuild '../dtsse-ccd-config-generator'
+//includeBuild '../rse-cft-lib'
```

Comment:
Do we need to revert this?

Reply:
No, commented out, they're useful when debugging CCD generator and/or CFTLib, but should never be live.